crud/common/utils.lua

```diff
@@ -1177,6 +1194,61 @@ function utils.storage_info_on_storage()
     return {status = "running"}
 end
 
+--- Initializes a storage function by its name.
+--
+--  It adds the function into the global scope by its name and required
+--  rights to a vshard storage user.
+--
+--  @function init_storage_call
+--
+--  @param string name a name of the function.
+--  @param function func the function.
+--
+--  @return nil
+function utils.init_storage_call(name, func)
+    dev_checks('string', 'function')
+
+    rawset(_G['_crud'], name, func)
+    name = utils.get_storage_call(name)
+
+    if type(box.cfg) ~= 'table' or box.cfg.read_only then
```

Comment:
Use `box.info.ro` instead of `box.cfg.read_only`. The first one always describe instance state (for example, one may mess with `box.cfg.read_only = false` in which case configuration will not be applied and the state will remain unchanged)

Reply:
But in this case, the configuration shows the user's expectations.

So if box.cfg.read_only == false, but box.info.ro == true and an error is thrown - I think this is better, then we just skip the configuration.

Reply:
I think it's a rule in Tarantool to check whether instance is ro/rw through box.info. And we want to know about whether we can or cannot write

Reply:
Replaced: `box.cfg.read_only` -> `box.info.ro`.